policies/lib/kubernetes.rego

```diff
@@ -169,3 +169,12 @@ volumes[volume] {
 	pods[pod]
 	volume = pod.spec.volumes[_]
 }
+
+#############
+# Functions #
+#############
+
+flag_contains_string(obj, key, value ) {
+  elems := [elem | contains(obj.command[i], key); elem := obj.command[i]]
```

Comment:
The failure that getting from the flux helm-operator.yaml is because this doesn't check if `command` attribute exists or not, so if the `command` doesn't exist, this function will be false, so where the function get called and start with not in the violation becomes true.